core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4509,6 +4510,7 @@ public static void _doScript(StaplerRequest req, StaplerResponse rsp, RequestDis
             }
 
             try {
+                LOGGER.log(Level.INFO, req.getRemoteAddr() + "|" + Jenkins.getAuthentication().getName() + "|ScriptConsole|" + Base64.getEncoder().encodeToString(text.getBytes()));
```

Comment:
Should we log channel information as well?

Reply:
Hi, not really sure what you're talking about. If you elaborate on channels and how they're used in the application I'll be able to share my thoughts. Is channel something like "we executed the ScriptConsole using the GUI vs the API"?